icommons_ext_tools/settings/base.py

```diff
@@ -79,7 +82,22 @@
 
 MANAGERS = ADMINS
 
-# DATABASES are defined in individual environment settings
+# DATABASES
+
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.oracle',
+        'NAME': SECURE_SETTINGS.get('db_default_name'),
```

Comment:
For an Oracle backend, would you mind sticking with the `django_db_*` secure keys as in icommons_tools (https://github.com/Harvard-University-iCommons/icommons_tools/blob/develop/icommons_tools/settings/base.py#L193-L206) as opposed to these `db_default_*` keys?  We have some deploy logic that would otherwise try to create a postgres db.